gamemodes/terrortown/gamemode/server/sv_addonchecker.lua

```diff
@@ -6,6 +6,7 @@
 
 ADDON_INCOMPATIBLE = 0
 ADDON_OUTDATED = 1
+TTT2_GMOD_MIN_VERSION = 241009
```

Comment:
```suggestion
TTT2_GMOD_MIN_VERSION = 240313
```

For better or worse my dev server setup wasn't up to date and this seems to be the march version, which is able to run the current master branch.

Reply:
Ahh, very nice thanks :)

I changed it to that version.